.editorconfig

```diff
@@ -7,7 +7,7 @@ indent_style = space
 indent_size = 2
 trim_trailing_whitespace = true
 charset = utf-8
-max_line_length = 80
+max_line_length = 120
```

Comment:
@riavalon @stevenkampen skipped my review. Do we need this? At the very least maybe this does not below here.